API/Services/ScoreService.cs

```diff
@@ -0,0 +1,50 @@
+﻿using API.Models;
+using System;
+using System.Collections.Generic;
+
+namespace API.Services
+{
+    public class ScoreService : IScoreService
+    {
+        public IDictionary<int, float> GetAspectScores(IList<QuestionOption> questionOptions)
+        {
+            IDictionary<int, float> aspectScores = new Dictionary<int, float>();
+            IDictionary<int, int> aspectCounts = new Dictionary<int, int>();
+
+            foreach (QuestionOption questionOption in questionOptions)
```

Comment:
Perhaps we should use `var` inside foreach?

Reply:
Done